### ISSUE TITLE
flutter_form_builder: ^3.13.3 Fails to compile app

### ISSUE THREAD

**Issue by fredwilliam** [danvick/flutter_form_builder#436]
../../snap/flutter/common/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_form_builder-3.13.3/lib/src/fields/form_builder_range_slider.dart:21:9: Error: 'RangeSemanticFormatterCallback' isn't a type.
  final RangeSemanticFormatterCallback semanticFormatterCallback;

class _RequestPageState extends State<RequestPage> {
  final GlobalKey<FormBuilderState> _fbKey = GlobalKey<FormBuilderState>();

  @override
  Widget build(BuildContext context) {
    return FormBuilder(
        key: _fbKey,
        child: SingleChildScrollView(child: Column(children: <Widget>[
          Text('Datetime Picker',
              style: TextStyle(fontSize: 20, color: Colors.black)),
          FormBuilderDateTimePicker(attribute: 'datetime_picker'),
          SizedBox(height: 20),
        ])));
  }
}

**Comment by chris-reynolds** [comment]
I have what is probably a related error building for ios:

```
    ../../../../development/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_form_builder-3.13.3/lib/src/fields/form_builder_range_slider.dart:21:9: Error: 'RangeSemanticFormatterCallback' isn't a type.
      final RangeSemanticFormatterCallback semanticFormatterCallback;
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
```

**Comment by chris-reynolds** [comment]
Ok, I see there is a rename fix in a pull-request.

**Comment by NiranjanShah** [comment]
What need to be done, can you please guide !!!!

**Comment by danvick** [comment]
This change cannot be done because it's in a pre-release version of Flutter. Anyone having this issue should consider using version 3.14.0-alpha.x of the package as suggested in the README.

**Comment by danvick** [comment]
Duplicate of #382
